CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## v0.4.1
+
+Conduit 0.4.1 builds on the telemetry work from 0.4.0, providing rich,
+Kubernetes-aware observability and debugging.
+
+* Web UI
+  * **New** Automatically-configured Grafana dashboards for Services, Pods,
+    ReplicationControllers, and Conduit Mesh Health. These dashboards also expose
+    new TCP-level stats.
```

Comment:
Break TCP out as a separate line item. This is big!

Reply:
i think this depends on whether we keep the `Telemetry` section below.